benchmark/galsim/GalsimBenchmark.jl

```diff
@@ -240,10 +265,13 @@ function main(; test_case_name=Nullable{String}())
         variational_parameters = DeterministicVI.infer_source(
```

Comment:
Rather than calling `infer_source`, would you call your `parallel_inference` routine here with `joint_infer = false`? That would be more symmetric.